modules/sno-about-sno-cluster-worker-nodes.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/nodes-sno-worker-nodes.adoc
+
+:_module-type: CONCEPT
+[id="sno-about-sno-cluster-worker-nodes_{context}"]
+= About worker nodes on single-node OpenShift clusters
+
+Worker nodes for single-node OpenShift clusters provide additional resources for your cluster. However, because there is only a single control plane node, ingress traffic is routed differently compared to standard cluster configurations with three control plane nodes.
+
+In a default single-node cluster configuration, all ingress traffic is routed through the control plane node by default. One of the benefits of installing single-node clusters is the simplicity of not having to deal with load-balancing and virtual IPs. Not having a load balancer installed by default increases cost savings and simplifies cluster deployment and configuration.
+
+When you add a worker node to the cluster, the Assisted Installer sets the `DefaultPlacement` field of the `IngressController` CR to be `ControlPlane`. This effectively routes all ingress traffic to worker nodes through the control plane node by default.
```

Comment:
>When you add a worker node to the cluster, the Assisted Installer sets the `DefaultPlacement` field of the `IngressController` CR to be `ControlPlane`. This effectively routes all ingress traffic to worker nodes through the control plane node by default.

nits:

- What's mentioned is a feature of the installer, not the assisted-installer

- defaultPlacement is a parameter of the ingress config CR, not the IngressController CR

I'm leaning towards omitting this entire paragraph, it's a bit of boring implementation details, as defaultPlacement is an immutable field users should not care about, it's just an internal way for the installer to communicate a general placement to the ingress controller

Reply:
removed the paragraph.